twine/utils.py

```diff
@@ -23,7 +23,7 @@
 import warnings
 import collections
 
-from requests.exceptions import HTTPError
+import requests.exceptions
```

Comment:
We were already munging styles, but now we're introducing just another disparate style here it seems.

Reply:
Sorry, maybe I'm missing something here? This is the same style as `import twine.exceptions`.